app/models/spree/adyen/hpp_source.rb

```diff
@@ -52,6 +52,10 @@ def can_cancel? payment
     payment.refunds.empty?
   end
 
+  def requires_manual_refund?
+    payment_method == "directEbanking" # aka sofort
```

Comment:
Could we make this a constant to account for other payment methods that may require a manual refund?